README.md

```diff
@@ -314,6 +314,16 @@ But follow [https://stripe.com/docs/billing/subscriptions/set-up-subscription](h
 
 - [ ] List all prices
 
+##### Products
+
+- [ ] Create a product
```

Comment:
Looks great, just mark this one as done as we have support for creating product as well, so it is accurate for the users looking at the readme :)

Reply:
Good catch :-)